src/booleans.js

```diff
@@ -1,61 +1,135 @@
 const negate = a => {
-  // your code here
+  if (a) {
+	return false
+}
+  else {
+	return true
+}		
```

Comment:
In this one all you need to do is to return the negate of the element. We know that ! represents not, so:
return !a